invenio_base/cmd.py

```diff
@@ -23,14 +23,22 @@
 
 """Application bootstraping."""
 
-from __future__ import absolute_import, print_function, unicode_literals
+from __future__ import absolute_import, print_function
+
+from pkg_resources import resource_filename
 
 import click
 
+from cookiecutter.main import cookiecutter
```

Comment:
Please check import sorting here using isort (see run-tests.sh) - not sure it's correct.